packages/openactive-broker-microservice/package.json

```diff
@@ -18,7 +18,7 @@
     "@openactive/data-model-validator": "^2.0.78",
     "@openactive/data-models": "^2.0.316",
     "@openactive/dataset-utils": "^1.0.1",
-    "@openactive/harvesting-utils": "github:openactive/harvesting-utils#9ae4474",
+    "@openactive/harvesting-utils": "github:openactive/harvesting-utils#9dad411",
```

Comment:
Newest version of harvesting-utils means:

- Types are accurate (which allowed me to see that onError was missing)
- Use more abstract `config.howLongToSleepAtFeedEnd` rather than `state.incompleteFeeds`